pandas/core/generic.py

```diff
@@ -7810,6 +7810,86 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         is not realigned. That is, use freq if you would like to extend the
         index when shifting and preserve the original data.
 
+        Examples
+        --------
+        Compute the difference between a column in a dataframe
```

Comment:
This first line isn't necessary - can simply delete

Reply:
Which first line? This one?  "Compute the difference between a column in a dataframe
        with grouped data, and its shifted version."

Reply:
I believe @WillAyd was referring to the "Compute the difference..." line.